src/backend/renderer/gles/mod.rs

```diff
@@ -501,7 +605,10 @@ impl GlesRenderer {
     /// - The renderer might use two-pass rendering internally to facilitate color space transformations.
     ///   As such it reserves any stencil buffer for internal use and makes no guarantee about previous framebuffer
     ///   contents being accessible during the lifetime of a `GlesFrame`.
-    pub unsafe fn new(context: EGLContext) -> Result<GlesRenderer, GlesError> {
+    pub unsafe fn new<C>(context: EGLContext, capabilities: C) -> Result<GlesRenderer, GlesError>
+    where
+        C: Into<Option<Vec<Capability>>>,
```

Comment:
I'd actually prefer the developer be explicit about what features are desired, if you want everything the driver supports then just pass the result of get_capabilities into this function.

Reply:
Why? To me that just seems like an unnecessary burden on most of the consumers.